ui/src/logs/containers/LogsPage.tsx

```diff
@@ -51,25 +51,29 @@ interface Props {
 interface State {
   searchString: string
   filters: Filter[]
+  liveUpdating: boolean
 }
 
 const DUMMY_FILTERS = [
-  {
-    id: '0',
-    key: 'host',
-    value: 'prod1-rsavage.local',
-    operator: '==',
-    enabled: true,
-  },
+  // {
```

Comment:
could you delete this commented code?

Reply:
I had commented this out just to get a feeling of how it looks but can uncomment.